### PR TITLE
Release 0.2.1

### PR DESCRIPTION
**Pull request by voropaevp**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -210,6 +210,7 @@ resource "aws_security_group_rule" "lb_egress_tcp_webserver" {
 # --- EC2: Auto-scaling group & Launch Configurations
 
 locals {
+
```
Comment (jbeemster) [inline main.tf]: Random newline?

File: main.tf [hunk 2]
```diff
@@ -220,6 +221,12 @@ locals {
     byte_limit    = var.byte_limit
     record_limit  = var.record_limit
     time_limit_ms = var.time_limit_ms
+
+    disable =  !tobool(var.telemetry_enabled)
+    userProvidedId = var.user_provided_id
+    moduleName = local.module_name
+    moduleVersion = local.module_version
+    autoGeneratedId = module.telemetry.0.auto_generated_id
```
Comment (jbeemster) [inline main.tf]: variables should be snakecase + missing terraform fmt on this file

File: templates/config.hocon.tmpl [hunk 3]
```diff
@@ -77,6 +77,13 @@ collector {
       timeLimit = ${time_limit_ms}
     }
   }
+  telemetry {
+    disable = ${disable}
+    userProvidedId = ${userProvidedId}
+    moduleName = ${moduleName}
+    moduleVersion = ${moduleVersion}
+    autoGeneratedId = ${autoGeneratedId}
```
Comment (jbeemster) [inline templates/config.hocon.tmpl]: Variable inputs should be snakecase

File: main.tf [hunk 4]
```diff
@@ -1,9 +1,9 @@
 locals {
   module_name    = "collector-kinesis-ec2"
-  module_version = "0.2.0"
+  module_version = "0.2.2"
```
Comment (jbeemster) [inline main.tf]: Sorry I got this wrong - was only relevant for GCP (which was why I deleted the comment but too late!)

**Comment by paulboocock** [comment]
Given the recent bumps to various libraries in the collector (like log4j and akka-http!), we've decide to update this outside of a "OS Distribution Release".

File: templates/config.hocon.tmpl [hunk 5]
```diff
@@ -77,6 +77,13 @@ collector {
       timeLimit = ${time_limit_ms}
     }
   }
+  telemetry {
```
Comment (jbeemster) [inline templates/config.hocon.tmpl:80]: Doesn't this need a collector endpoint for the telemetry?

Comment (paulboocock) [reply templates/config.hocon.tmpl:80]: Updated to use the same url as the telemetry module

**Review by jbeemster — APPROVED** [review]
LGTM again assuming you have spun up the module and tested it out.

**Pull request merged**